.env-template

```diff
@@ -1,2 +1,5 @@
 GITHUB_TOKEN=
-STORAGE_KEY=
+STORAGE_KEY="DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1;"
+WEEKLY_RETENTION=1
+GITHUB_ORG=equinor
+BLOB_CONTAINER=github-archives
```

Comment:
Good idea to put them in `.env`.

Maybe add a comment over `STORAGE_KEY`, that it is the default secret, and add a link to where one can find more information (same URL as in `decker-compose.override.yml`).
That way, a new developer won't be surprised / shocked to find a hard-coded secret.

Reply:
fixed